akka-stream/src/main/scala/akka/stream/Materializer.scala

```diff
@@ -25,6 +25,7 @@ import akka.event.LoggingAdapter
  *
  * Not for user extension
  */
+@implicitNotFound("A Materializer is required.  You may want to have the ActorSystem in implicit scope")
```

Comment:
how about:
A Materializer is required to run the RunnableGraph. You may want to have the ActorSystem in implicit scope, such as `implicit val actorSystem = ....`